src/_tutorials/server/get-started.md

```diff
@@ -139,21 +136,25 @@ Dart code to optimized native machine code.
 Use the `dart2aot` tool to AOT compile the program to machine code:
 
 ```terminal
-> dart2aot bin/main.dart bin/main.dart.aot
+$ dart2aot bin/main.dart bin/main.dart.aot
 ```
 
 To run the compiled program, use the Dart runtime (`dartaotruntime`):
 
 ```terminal
-> dartaotruntime bin/main.dart.aot
+$ dartaotruntime bin/main.dart.aot
+Hello world: 21!
 ```
 
 Notice how the compiled program starts instantly, completing quickly:
 
 ```terminal
-> time dartaotruntime bin/main.dart.aot
+$ time dartaotruntime bin/main.dart.aot
+Hello world: 21!
 
-real	0m0.032s
+real	0m0.016s
```

Comment:
Wow, your computer's a lot faster than mine! Right now I'm getting this (with 2.5.0 stable, but 2.6.0-dev.1.0 is about the same):

```
Hello world: 21!

real	0m0.064s
user	0m0.024s
sys	0m0.038s
```

I wonder why there's such a difference between your and my computer. I hesitate to use your numbers if mine are more representative. (And vice versa...)

Reply:
Keeping the new numbers, as agreed out of band.